packages/genesys-spark-components/src/components/stable/gux-popover/gux-popover.tsx

```diff
@@ -251,6 +251,7 @@ export class GuxPopover {
           'gux-hidden': !this.isOpen,
           'gux-popover-wrapper': true
         }}
+        role="dialog"
```

Comment:
I don't think a popover is a dialog. Have a read of this and see if you agree or not https://bootcamp.uxdesign.cc/popups-dialogs-tooltips-and-popovers-ux-patterns-2-939da7a1ddcd

Reply:
@daragh-king-genesys  I agree with you. After researching more I don't think dialog is the right choice. It seems like the dialog role would be a good choice if we had a smaller window that was blocking in some sort to direct user to that dialog and make it the main focus of the page perhaps. popover-list seems to be a little more passive than a dialog and not as blocking. I'm trying to figure out what would be a better choice than dialog. 

Some potential role options I found are:

1) tooltip - I'm not sure about this one. So at first I thought tooltips were good for additional content that would show on hover, but from researching it looks like you can have tooltips that show on click too. I don't think tooltip makes sense for us because we're always going to show a list of some sort, and not just additional text, which is what I'd think the tooltip role would be good for.
1) list - I think this is more of a general list (e.g. `ul`) that has plain text in each item
1) listbox - this seems to be used for elements like a `select` where you have different options to choose from. Could be used potentially but we won't always necessarily have selectable list items and could have just plain text.
1) menu - this seems to be for actionable list items (e.g. clicking a button to trigger an action). I don't think this one makes sense since you can have actionable or non-actionable items in popover-list.

Since popover-list can have any sort of list elements then I'm thinking maybe a more generic `list` role might be good. What do you think?